CONTRIBUTING.md

```diff
@@ -7,11 +7,20 @@ practices solidify.
 
 1. Copy the sample env file
 
-    `cp .env.sample .env.local`
+   `cp .env.sample .env.local`
 
 1. Modify variables in .env.local, e.g. uncommenting the `EXPERIMENTER_API_CALL`
    line will switch from the default of live experiments to preview experiments
 
+## Linting
+
+To correctly run [Prettier](https://prettier.io/docs/en/install) as a pre-commit hook, set up [husky](https://github.com/typicode/husky):
+
+```
+npm i
+npx husky init
+```
+
```

Comment:
This section is the only part I added to CONTRIBUTING, the rest was the pre-commit linting.